src/msg/async/net_handler.cc

```diff
@@ -44,7 +44,7 @@ int NetHandler::create_socket(int domain, bool reuse_addr)
   /* Make sure connection-intensive things like the benchmark
    * will be able to close/open sockets a zillion of times */
   if (reuse_addr) {
-    if (::setsockopt(s, SOL_SOCKET, SO_REUSEADDR, &on, sizeof(on)) == -1) {
+    if (::setsockopt(s, SOL_SOCKET, SO_REUSEADDR, nullptr, 0) == -1) {
```

Comment:
I don't think this is right.  From teh man page,
<pre>
       Most socket-level options utilize an int argument for optval.  For setsockopt(), the argument  should  be  nonzero  to  enable  a  boolean
       option, or zero if the option is to be disabled.
</pre>
We're enabling the option, so I think we need to pass the (non-zero) int value.